templates/magazine/_list.html.twig

```diff
@@ -63,7 +63,7 @@
                     <tbody>
                     {% for magazine in magazines %}
                         <tr>
-                            <td>{{ component('magazine_inline', { magazine: magazine, stretchedLink: true, showAvatar: true, fullName: true}) }}</td>
+                            <td>{{ component('magazine_inline', { magazine: magazine, stretchedLink: true, showAvatar: true}) }} {% if magazine.isAdult %}<small class="badge danger">18+</small>{% endif %}</td>
```

Comment:
I thought about adding the adult badge to magazine inline when full name was false (it only appears when full name is true) but since at the moment it's just this component, didn't seem worth it adding something that would be unset/false in every single other instance, so instead just appended it here